portfolio/src/main/webapp/index.html

```diff
@@ -49,25 +49,25 @@ <h1>Michaela Murray's Portfolio</h1>
         <div class="mdc-touch-target-wrapper">
           <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank">
             <button class="mdc-button mdc-button--outlined">  
-	       <span class="mdc-button__ripple">Resume</span> 
-	    </button>
+	            <span class="mdc-button__ripple">Resume</span> 
+	        </button>
           </form>
         </div>
 	<div id="icons">
             <ul> 
             <li> 
-                <a href="https://github.com/mmurray22" target="_blank">
-                    <img id="icon" src="/images/logo-github.svg">
+                <a href="https://github.com/mmurray22" target="_blank" alt="Github icon">
```

Comment:
The `alt` attribute should apply to the `img` element. The first two links had the alt applied to the anchor, instead.

Reply:
thanks! updated!